frontend/scripts/react-components/tool/column-selector/column-selector.component.jsx

```diff
@@ -37,18 +39,17 @@ function ColumnSelector(props) {
   return (
     <div
       className="c-column-selector"
-      style={{ left: group * sankeyColumnsWidth + group * gapBetweenColumns - 8 }}
+      style={{ left: position * sankeyColumnsWidth + position * gapBetweenColumns - 8 }}
     >
       {content}
     </div>
   );
 }
 
 ColumnSelector.propTypes = {
-  group: PropTypes.number.isRequired,
```

Comment:
group is the name in the column object do you prefer having a local variable name instead?

Reply:
I don't get this 🤷‍♂